session-9/src/pages/Home/Home.js

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment, useState, useRef } from "react";
+import Card from "../../components/Card/Card";
+import Header from "../../components/Header/Header";
+import SelectBox from "../../components/SelectBox/SelectBox";
+import useGitHubMembersByCity from "../../components/../hooks/useGithubMembersByCity";
+import "./Home.css";
+
+const Home = () => {
+  const ref = useRef();
+  const [searchText, setSearchText] = useState("");
+
+  const [selectedState, setSelectedState] = useState("");
+  const [selectedCity, setSelectedCity] = useState("");
+
+  const { data, loading } = useGitHubMembersByCity(selectedCity);
+
+  console.log(data);
+
+  const handleSearchText = () => {
+    setSearchText(ref.current.value);
+  };
+
+  return (
+    <Fragment>
+      <Header reference={ref} handleSearchText={handleSearchText} />
+      <SelectBox
+        selectedState={selectedState}
+        setSelectedState={setSelectedState}
+        selectedCity={selectedCity}
+        setSelectedCity={setSelectedCity}
+      />
+      <div className="card-list">
+        {loading && <div>Loading...</div>}
```

Comment:
set the loading state to false where it should be and remove it from Line 34. Can also use the ternary operator.